pkg/minishift/addon/parser/addon_parser.go

```diff
@@ -180,12 +180,19 @@ func (parser *AddOnParser) parseCommands(scanner *bufio.Scanner) ([]command.Comm
 }
 
 func createMetaData(header []string) (addon.AddOnMeta, error) {
-	regex, _ := regexp.Compile(`^# ?(.*):(.*)`)
+	regex, _ := regexp.Compile(`^# ?([a-zA-Z-]*):(.*)`)
```

Comment:
I would prefer regular expressions are defined as a constant and get a name, as this makes their intention of use much clearer.